README.md

```diff
@@ -1,20 +1,22 @@
 # Contents of this repository
 This repository contains software tools which allow to compare spectra using the Wasserstein distance and estimate relative abundances of molecules from the spectrum by minimizing the Wasserstein distance. 
 
-The tools are distributed as a Python3 package called `masserstein`. Basic functionality is also available as a set of commandline applications: `WSDistance` to compute the Wasserstein distance and `WSDeconv` to estimate proportions. 
+The tools are distributed as a Python3 package called `masserstein`. The package contains functions for generating spectra from chemical formulas, basic processing (centroiding, smoothing), and fitting combinations of reference spectra of analytes to a spectrum of their mixture.  
+Examples of use of the package functions are shown in the tutorials available on this repository.  
```

Comment:
features?

Reply:
Ok, this kind of functions. I suggest simplifying as below because I have a problem with them.

"The tools are distributed as a Python3 package called `masserstein`. The package can be used for generating spectra from chemical formulas, basic processing (centroiding, smoothing), and fitting combinations of reference spectra of analytes to a spectrum of their mixture.  
Examples of package usage are shown in the tutorials available on this repository."